wiki/tests.py

```diff
@@ -63,10 +64,11 @@ def test_preview_save(self):
         example2['content'] = 'Something 2'
         response = c.post(reverse('wiki:edit', kwargs={'path': ''}), example2)
         message = c.cookies['messages'].value if 'messages' in c.cookies else None
+        decoded_message = CookieStorage(response)._decode(message)[0].message
```

Comment:
`Django 3.2` updated the cookie message handling. 
[CookieStorage](https://docs.djangoproject.com/en/4.0/ref/contrib/messages/#django.contrib.messages.storage.cookie.CookieStorage) now stores messages in the [RFC 6265](https://tools.ietf.org/html/rfc6265.html) compliant format.